source/reference/minio-server/settings/console.rst

```diff
@@ -226,65 +228,125 @@ Set the appropriate environment variable or configuration settings according to
 
       .. envvar:: MINIO_BROWSER_HSTS_SECONDS
 
-	 The ``max_age`` the configured policy remains in effect, in seconds. 
-	 Defaults to ``0``.
+         The ``max_age`` the configured policy remains in effect, in seconds.
+         Defaults to ``0``, disabled.
+         You **must** configure a *non-zero* duration to enable the ``Strict-Transport-Security`` header.
 
-      .. code-block:: shell
-         :class: copyable
+         .. code-block:: shell
+            :class: copyable
 
-         set MINIO_BROWSER_HSTS_SECONDS=31536000
+            set MINIO_BROWSER_HSTS_SECONDS=31536000
 
    .. tab-item:: Configuration Settings
       :sync: config
 
+      The following configuration settings require a service restart to take effect.
+      You can do this with :mc-cmd:`mc admin service restart`.
+
       .. mc-conf:: browser hsts_include_subdomains
          :delimiter: " "
-		     
-      .. code-block:: shell
 
-         mc admin config set browser \
-            hsts_include_subdomains="on" \
-            [ARGUMENT=VALUE ...]
-	    
+         Set to ``on`` to also apply the configured HSTS policy to all MinIO Console subdomains.
+         Defaults to ``off``.
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin config set browser \
+               hsts_include_subdomains="on" \
+               hsts_seconds="31536000" \
+               [ARGUMENT=VALUE ...]
+
       .. mc-conf:: browser hsts_preload
          :delimiter: " "
-     
-      .. code-block:: shell
 
-         mc admin config set browser \
-            hsts_preload="on" \
-            [ARGUMENT=VALUE ...]
-	    
+         Set to ``on`` to direct the client browser to add the MinIO Console domain to its HSTS preload list.
+         Defaults to ``off``.
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin config set browser \
+               hsts_preload="on" \
+               hsts_seconds="31536000" \
+               [ARGUMENT=VALUE ...]
+
       .. mc-conf:: browser hsts_seconds
          :delimiter: " "
-        	
-      .. code-block:: shell
 
-         mc admin config set browser \
-            hsts_seconds="0" \
-            [ARGUMENT=VALUE ...]
+         The ``max_age`` the configured policy remains in effect, in seconds.
+         Defaults to ``0``, disabled.
+         You **must** configure a *non-zero* duration to enable the ``Strict-Transport-Security`` header.
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin config set browser \
+               hsts_seconds="31536000" \
+               [ARGUMENT=VALUE ...]
+
+
+Examples
+++++++++
+
+The following examples show the rendered header for the given configuration settings.
+The equivalent environment variables generate the same result.
+
+``hsts_seconds``
+
+  .. code-block:: shell
+     :class: copyable
+
+     mc admin config set play browser hsts_seconds=500
```

Comment:
```suggestion
     mc admin config set ALIAS browser hsts_seconds=500
```

Reply:
👍🏻 I've got that in the edit I'm working on right now 😄